data/src/main/java/camp/nextstep/edu/github/data/di/NetworkModule.kt

```diff
@@ -0,0 +1,22 @@
+package camp.nextstep.edu.github.data.di
+
+import camp.nextstep.edu.github.data.service.GithubService
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object NetworkModule {
```

Comment:
```suggestion
internal object NetworkModule {
```
구현체는 internal 접근자를 붙여보면 어떨까요?